src/site/stages/build/plugins/create-header-footer.js

```diff
@@ -51,31 +19,19 @@ function createHeaderFooterData(buildOptions) {
       megaMenuData,
     };
 
-    const serialized = JSON.stringify(headerFooter, null, 4);
-
-    const drupalMenu = DRUPALS.PREFIXED_ENVIRONMENTS.has(buildOptions.buildtype)
-      ? replaceWithDrupalLinks(headerFooter, files)
-      : headerFooter;
-    const drupalMenuSerialized = JSON.stringify(drupalMenu, null, 4);
+    const stringified = JSON.stringify(headerFooter);
 
-    Object.keys(files).forEach(file => {
-      if (files[file].isDrupalPage) {
-        // eslint-disable-next-line no-param-reassign
-        files[file].headerFooterData = drupalMenuSerialized;
-      } else {
-        // eslint-disable-next-line no-param-reassign
-        files[file].headerFooterData = serialized;
-      }
```

Comment:
There was a bunch of old code here used when only a small segment of the site was building from Drupal. On those pages, the build would inject the  headerFooter data from Drupal but on the other pages (from vagov-content) we would continue to use the headerFooter data we had on GitHub. Obviously, the whole site now uses the headerFooter data from Drupal, so I just removed the conditions to flatten the logic.